circle.yml

```diff
@@ -118,6 +125,17 @@ test:
         [ "$(node -p 'require("./status.json").completed')" != false ] && break
       done
 
+    # Wait for screenshots to be ready
+    - while [ ! -e ~/screenshots ]; do sleep 1; done:
```

Comment:
wanna rename to something slightly more descriptive, like `~/screenshots_are_ready` (by analogy with `~/sauce_is_ready`)?